README.md

```diff
@@ -39,18 +40,27 @@ Maybe it doesn't help that the first option explained in `man tar` is:
    and usually not even then as the default block size of 20 records (10240 bytes) is very common.
 ```
 
-Surely people could benefit from simplified man pages
-focused on practical examples.
+There seems to be room for simpler help pages, focused on practical examples.
 How about:
 
 ![screenshot of the tldr-node-client displaying the tldr page for the tar command](images/screenshot.png)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX, Linux, macOS, SunOS and Windows commands.
+for the most common UNIX, Linux, macOS, SunOS and Windows command-line tools.
+
+## How do I use it?
 
-## Clients
+A popular and convenient way to access these pages on your computer
+is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-client),
+which is supported by the tldr-pages project maintainers:
 
-You can access these pages on your computer using one of the following clients:
+    npm install -g tldr
+
+That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
+just like you would write `man tar` to show its manpage.
```

Comment:
Hi @waldyrious , what about?

```suggestion
That way you can write `tldr tar` in the terminal and you will see the most common situations where such a command is used. Isn't that great?
```

Reply:
Can you explain a bit why you prefer this version?

Note that we already explain in the previous section what a tldr page is, so it's redundant to add yet another take at that here; and besides, with your suggestion we lose the parallel with manpages, which we [agreed in #1192](https://github.com/tldr-pages/tldr/issues/1192#issuecomment-629808084) that we should keep.

Reply:
Hi @waldyrious I totally forgot about the parallel with manpages and didn't pay attention that tldr was explained before

my bad :grimacing: